app/src/main/cpp/src/components/transport_manager/src/bluetooth_le/ble_server.cc

```diff
@@ -5,8 +5,6 @@
 SDL_CREATE_LOG_VARIABLE("Ble_Server")
 
 namespace{
-    const char* socket_name =  "./localBleWriter";
-
     // To align with requested MTU size
     const int buffer_size = 185;
```

Comment:
For now there is no need in aligning with MTU size, so please increase buffer size to reasonable maximum value

Reply:
done in 66d56ca414ac2d17dea823a2186a8d3f2569ec83